src/server/saved_objects/routes/bulk_get.js

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi';
+
+export const createBulkGetRoute = (prereqs) => ({
+  path: '/api/saved_objects/_bulk_get',
```

Comment:
can you add a comment on the underscore, is this meant to be a private endpoint?